docs/docs/channels/email/emailjs.md

```diff
@@ -1,23 +1,27 @@
-# EmailJS
+# Email.js
 
-[EmailJS](https://www.emailjs.com/) helps to send emails using client-side technologies only. No server is required – just connect EmailJS to one of the supported email services, create an email template, and use our Javascript library to trigger an email. You can use EmailJS to send the content of a form filled by your users directly to your email address, using a simple connection provided by EmailJS.
+You can use the [Email.js](https://www.emailjs.com/) provider to send transactional emails to your customers using the Novu Platform with a single API to create multi-channel experiences.
 
 ## Getting Started
 
-You have to simply create an account on EmailJS and verify your email address after that you can use API or CDN provided by EmailJS to send a response of form to your email address.
+To use the Email.js channel, you will need to create a Email.js account and add your API key to the Email.js integration on the Novu platform.
 
-## How to connect a form to your email?
+## Find the API Key
 
-- Create a simple HTML form.
-- Create an account on EmailJS and verify your email.
-- Create a new service and do all the settings.
-- Create a connection using CDN or API.
+To find your Email.js API key, log into your Email.js account and navigate to the [API Keys](https://dashboard.emailjs.com/admin/account) page.
+It is suggested that you create a new API key for use with Novu. To successfully send emails, you will need to add the following permissions to your API key:
 
-## Integrate EmailJS with Novu
+## Authenticate your sender identity
+
+All email services require some sort of authentication to send the emails on your behalf. That makes it a bad idea to use them directly from client-side – revealing your password or your secret keys will allow anyone to send emails on your behalf.
+
+Email.js keeps your authentication details on the server-side, and the client-side code just triggers a predefined email template, similarly to how any client-server application is working.
```

Comment:
This bit belongs to this area: https://www.emailjs.com/docs/faq/cant-i-use-services-like-sendgrid-or-mandrill-directly/
It tries to explain the benefits of using Email.js when dealing with other services. To be honest, has little relation to the purpose of this documentation that is to teach to the users how to integrate this provider in Novu.